app/components/widgets/forms/image-upload.js

```diff
@@ -22,6 +23,12 @@ export default Component.extend({
     return this.needsCropper && !this.selectedImage.includes('http');
   }),
 
+  routing: Ember.inject.service('-routing'),
+
+  isProfileRoute: Ember.computed('routing.currentRouteName',  function() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use `import { computed } from '@ember/object';` instead of using Ember.computed](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4869050)